src/main/java/org/prebid/server/auction/gpp/AuctionGppService.java

```diff
@@ -0,0 +1,105 @@
+package org.prebid.server.auction.gpp;
+
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Regs;
+import com.iab.openrtb.request.User;
+import org.apache.commons.lang3.ObjectUtils;
+import org.prebid.server.auction.gpp.model.GppContext;
+import org.prebid.server.auction.gpp.model.GppContextCreator;
+import org.prebid.server.auction.model.AuctionContext;
+import org.prebid.server.util.ObjectUtil;
+
+import java.util.List;
+import java.util.Optional;
+
+public class AuctionGppService {
```

Comment:
What are differences between this class and AmpGppService? I don't see them at first glance.